xml_inscriptions/metadata_only/NJ.Princ.PU.AM.G.3775-I67.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- v. 20. 2019-10-04 EM  -->
+<!--<?oxygen RNGSchema="http://cds.library.brown.edu/projects/usepigraphy/schema/exp-epidoc.rng" type="xml"?>
+--><TEI xmlns="http://www.tei-c.org/ns/1.0" xmlns:xi="http://www.w3.org/2001/XInclude">
+    <teiHeader>
+
+        <fileDesc>
+
+            <titleStmt>
+                <title>NJ.Princ.PU.AM.G.3775-I67</title>
+            </titleStmt>
+
+            <publicationStmt>
+                <authority>Brown University</authority>
+                <idno type="USEpigraphy" xml:id="NJ.Princ.PU.AM.G.3775-I67">Repeat USEP ID</idno>
+            </publicationStmt>
+
+            <sourceDesc>
+                <!--  *************************************  <msDesc>  ********************************* -->
+                <msDesc>
+                    <msIdentifier>
+                        <region>NJ</region>
+                        <settlement>Princ</settlement>
+                        <institution>PU</institution>
+                        <repository>AM</repository>
+                        <idno>3775-I67</idno>
+                    </msIdentifier>
+
+                    <!--  *************************************  <msContents>  *********************************    -->
+                    <msContents>
+                        <textLang mainLang="grc" otherLangs=""/>
+                        <!-- arc, grc, he, lat, und (unknown),  zxx (non-linguistic content) -->
+                        <msItem class="#funerary">
```

Comment:
If an inscription is a decree, should it be #document.decree?

Reply:
Not in this case, since the vocab doesn't have something specific for decree. Perhaps document.public, but if we're not sure best to just leave it as document.